guide/creating-your-bot/README.md

```diff
@@ -153,6 +155,29 @@ client.once('ready', () => {
 // Login to Discord with your client's token
 client.login(token);
 ```
+:::
+::: code-group-item .env file
+```js
+const { Client, GatewayIntentBits } = require('discord.js');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const { token } = process.env["DISCORD_TOKEN"]
```

Comment:
`process.env["DISCORD_TOKEN"]` would be a string, you can't destructure `token` out of it 
and besides, discord.js defaults the client token to that (`process.env.DISCORD_TOKEN`, if set), there's no need to pass it

Reply:
Yeah I figured that out like right before Is aw that reply lol. I had to find out the hard way when my code didnt work 💀